Cargo.lock

```diff
@@ -3317,6 +3317,7 @@ dependencies = [
  "rustc_parse",
  "rustc_session",
  "rustc_span",
+ "rustc_target",
```

Comment:
`rustc_target` is already an indirect dependency via `rustc_session`, so this does not really impact the crate graph.